lib/test-worker.js

```diff
@@ -34,73 +30,40 @@ sourceMapSupport.install({
 		if (sourceMapCache[source]) {
```

Comment:
I didn't modify this line, the only modification I made was the one discussed below (saving to disk in a different thread and loading here).